.devcontainer/devcontainer.json

```diff
@@ -0,0 +1 @@
+{ "image": "ubuntu:22.04" }
```

Comment:
This is just to test building on linux similar to the pipeline to see what missing deps there are